maintenance_request_report/views/report_maintenance_request.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2020 - TODAY, Marcel Savegnago - Escodoo
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+    <data>
+        <template id="report_maintenance_request_document">
+            <t t-call="web.external_layout">
+                <t t-set="doc" t-value="doc.with_context(lang=doc.user_id.partner_id.lang)" />
+
+                <div class="page">
+                    <div class="oe_structure" />
+
+                    <h2 class="mt16">
+                        <t>
+                            <span>Maintenance Request #</span>
```

Comment:
I think # shouldn't be required. If you need the # you can modify the sequence, but if you made MR0001, now it will be shown #MR0001 and it has no sense for me

Reply:
For now I removed the dependency on manitenance_request_sequence